samples/net/zperf/overlay-wifi-nm-hostap.conf

```diff
@@ -0,0 +1,19 @@
+# For Wi-Fi STA mode
```

Comment:
Couldn't we just use the Wi-Fi snippet used by other samples?
https://github.com/zephyrproject-rtos/zephyr/blob/main/snippets/wifi-ipv4/wifi-ipv4.conf
There're slight differences in the configuration, but if we need more resources for this sample we could perhaps adjust it.

Reply:
Ah right, forgot about this snippet, we can use that, thanks.

Reply:
```
+# Zperf TCP=6, UDP=2, Others=1
+# WPA supplicant: Global ctrl_iface: 2, per interface: 2
+CONFIG_ZVFS_OPEN_MAX=20
+CONFIG_NET_SOCKETS_POLL_MAX=20
+CONFIG_ZVFS_POLL_MAX=20
```
How do we deal with these options? I can move to board specific, but these are common to wi-fi but only need when wi-fi and zperf are used together?

Reply:
The snippet already sets `CONFIG_ZVFS_OPEN_MAX=24`. As for `CONFIG_ZVFS_POLL_MAX`, snippet sets this to 9, same as the default `prj.conf` in the zperf sample. Do we really need to have this set to 20? I'm not sure if I understand why would we need more than the default from prj.conf specifically for Wi-Fi case. But if that really the case - I don't see a problem with increasing the config value in the snippet.

Regarding `CONFIG_NET_SOCKETS_POLL_MAX` - it's deprecated and redundant with `CONFIG_ZVFS_POLL_MAX`, we should just drop it.

Reply:
I have dropped that commit, I will run more tests with snippet and get back to you on the socket settings. Thanks.

Reply:
The snippet works as is, no need to update, thanks.

Reply:
I had to update the snippet too, earlier I had the increased settings in the board file.